lessons/lesson3.md

```diff
@@ -0,0 +1,11 @@
+## Lesson 3: Custom CI/CD workflow 
+
+The default implementation of the CI/CD workflow for Project Firefly Applications relies on GitHub capabilities. 
+Unfortunately, GitHub actions are not supported on GitHub Enterprise yet.
+ 
+In the case you can't use the GitHub Actions, we recommend implementing a custom CI/CD workflow with focus on two main aspects:
+
+* The [CLI](https://github.com/adobe/aio-cli) is the official tool to manage the Project Firefly Application development lifecycle from bootstrapping to deployment, and can be used within a CI/CD workflow for automation purpose.
+* Security is a key requirement, and any alternative CI/CD workflow should propose a solid secret management solution to store the credentials required to deploy a Project Firefly Application against a specific **Workspace**.
```

Comment:
can we have more details for setting up custom env vars here? Ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files
(as a side note, the `env` directive has been deprecated: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/)

Reply:
Yes pls check https://github.com/AdobeDocs/adobeio-codelabs-ci-cd/pull/4/commits/cd6cc343dbe744113631e904d0a45d3737e721ec thx !